docs/UserGuide.md

```diff
@@ -65,129 +65,125 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/Johnson p/83918273 e/johnson@gmail.com t/Hustlers`
 
-### Listing all persons : `list`
 
-Shows a list of all persons in the address book.
+### Deleting a person: `del`
 
-Format: `list`
+Delete a person from our contact list.
+
+Format: `del INDEX`
+
+Examples:
+* `del 1` deletes the first person in the contact list.
 
-### Editing a person : `edit`
 
-Edits an existing person in the address book.
+### Adding a tag : `tag-add`
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Add tags to a selected person from our contact list.
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+Format: `tag-add INDEX  TAG_NAME`
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `tag-add 1 Public Relations` adds the tag `Public Relations` to the first person in the contact list.
 
-### Locating persons by name: `find`
 
-Finds persons whose names contain any of the given keywords.
+### Deleting a tag : `tag-del`
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Add tags to a selected person from our contact list.
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+Format: `tag-del INDEX  TAG_INDEX`
 
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+* `tag-del 1 1` deletes the first tag from the first person in the contact list.
 
-### Deleting a person : `delete`
 
-Deletes the specified person from the address book.
+### Locating persons by keyword : `find`
 
-Format: `delete INDEX`
+Find persons matching any of the given keywords from our contact list.
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+Format: `find KEYWORD [MORE_KEYWORDS]`
```

Comment:
Should the format for this be -> find [n/NAME] [t/TAG] instead?

Reply:
But I think we also need to figure out a way to show that at least one of the optional fields should be present

Reply:
In that case perhaps instead the format can be -> find n/NAME [t/TAG], so the tag (if known) will only be used to differentiate people with the same name?

Reply:
then finding/filtering by a given tag specifically will be a separate command?

Reply:
I was thinking of using the find command to allow the users to either find by name or find by tag. Do you think it's better to separate the commands?

Reply:
Yeap sounds good. I will tag everyone here. @AAlghrairy @juayhee @Rye-Catcher

Reply:
I think @simjunheng way is better, but I think it would be better if it were written as `find [KEYWORD] [t/TAG]`.

Reply:
I think it is good to have some way to find in tags or names. AAlghrairy's suggestion is pretty good. We can have `t/` or `n/` to specify a `find` command to only find in tags or names. So that it is easier later if we want to find keywords in both names and tags, or find keywords in notes, address e.t.c.

Reply:
Okay! I will take in your suggestions and change the portion accordingly once the find [t/TAG] is pushed. Shall we merge this first?

Reply:
I think it should be OK to merge for now